docs/sftt/sftt-api-spec.md

```diff
@@ -1230,3 +1361,298 @@ Returns a list of the top 100 team's names and the blocks those teams counted, i
 ##### `400 BAD REQUEST`
 
 If the request was malformed.
+
+## Sites Router
+
+The sites router is used to handle all the sites and create new ones. A site can be either a planting site without a tree or there can be a tree present. They are differentiated by the field `tree_present` in the `site_entries` table. 
+
+### Add a Site
+
+`POST api/v1/protected/sites/create`
+
+Used to create a new site. Will create two entries in the database. One in the `sites` table to record the permanent information (location, address, block_id) and one in the `site_entries` table to record the state of the site (species, foliage, leaning, trash, etc.). Every field besides `block_id`, `lat`, `lng`, `city`, `zip` and `address` is allowed to be `NULL`.
+
+#### Request Body
+
+```json
+{
+    "block_id": INT,
+    "lat": LONG,
+    "lng": LONG,
+    "city": STRING,
+    "zip": STRING,
+    "address": STRING,
+    "tree_present": BOOLEAN,
```

Comment:
I know it's annoying because there are a ton of fields but if they can be null you should write their type as:
```
"tree_present": BOOLEAN | NULL
```